Playwright.Dockerfile

```diff
@@ -9,5 +9,3 @@ WORKDIR /app
 COPY . .
 
 RUN make install
-
-CMD ["pnpm", "run", "dev"]
```

Comment:
Docker container by default will stop if you don’t have anything in CMD
so let’s write there
CMD ["/bin/bash"]

it should help to fix exiting and restarting the playwright container